src/libs/antares/benchmarking/studyinfo.cpp

```diff
@@ -1,12 +1,53 @@
 #include "studyinfo.h"
-#include <antares/study.h>
+#include <antares/exception/LoadingError.hpp>
+#include "antares/study/area/area.h"
 
-namespace Benchmarking {
-  StudyInfo::StudyInfo() = default;
+using namespace Antares::Data;
 
+namespace Benchmarking 
+{
+	StudyInfo::StudyInfo(const Antares::Data::Study& study) : study_(study)
+	{}
 
-  void StudyInfo::initializeFromStudy(const Antares::Data::Study&)
-  {
+	void StudyInfo::collect()
+	{
+		getNbPerformedYears();
+		getEnabledThermalClusters();
+	}
 
-  }
+	void StudyInfo::flush(Yuni::String& filePath)
+	{
+		if (!outputFile_.openRW(filePath))
+		{
+			throw Antares::Error::CreatingStudyInfoFile(filePath);
+		}
+
+		outputFile_ << "Number of performed years" << "\t" << nbPerformedYears_ << "\n";
+		outputFile_ << "Number of enabled thermal clusters" << "\t" << nbEnabledThermalClusters_ << "\n";
```

Comment:
Since this data is meant to be parsed for statistical analysis, I suggest making it less human-readable, and more easily parsable. Avoid spaces, use short names, etc.

Reply:
I agree. Could you be more specific about this file format ?